wpm/config.py

```diff
@@ -87,19 +93,21 @@ def __getattr__(self, name):
         try:
             return convert(value)
         except ConfigError as e:
-            raise ConfigError("Error in .wpmrc section %r option %r: %s" %
+            raise ConfigError("Error in wpmrc section %r option %r: %s" %
                     (self.section, name, e))
 
 
 class Config(object):
-    """Contains the user configuration, backed by the .wpmrc file."""
+    """Contains the user configuration, backed by the wpmrc file."""
     # pylint: disable=too-many-public-methods
 
     config = None
 
     def __init__(self):
         Config.config = configparser.ConfigParser()
-        self.filename = os.path.expanduser("~/.wpmrc")
+        config_directory = get_config_directory()
+        self.filename = os.path.join(config_directory, "wpmrc")
+        os.makedirs(config_directory, exist_ok=True)
```

Comment:
I don't like creating this directory.

I'd rather want `get_config_directory` to (1) check if you already have `~/.wpmrc` and just use that if people already have it there. If not, check that `XDG_CONFIG_HOME` is set and points to an _existing_ directory, then return that if so.